src/Sylius/Behat/Context/Setup/PaymentContext.php

```diff
@@ -151,29 +170,42 @@ public function itHasInstructions(PaymentMethodInterface $paymentMethod, $instru
      */
     public function theStoreHasPaymentMethodNotAssignedToAnyChannel($paymentMethodName)
     {
-        $this->createPaymentMethod($paymentMethodName, 'PM_'.$paymentMethodName, 'Payment method', false);
+        $this->createPaymentMethod($paymentMethodName, 'PM_'.$paymentMethodName, 'Payment method', 'Offline', false);
     }
 
     /**
      * @param string $name
      * @param string $code
-     * @param bool $addForCurrentChannel
+     * @param string $gatewayFactory
      * @param string $description
+     * @param bool $addForCurrentChannel
+     * @param int|null $position
      *
      * @return PaymentMethodInterface
      */
-    private function createPaymentMethod($name, $code, $description = '', $addForCurrentChannel = true, $position = null)
-    {
+    private function createPaymentMethod(
```

Comment:
There are 5 places here, where this method is used. With the introduction of example factory, do you think that removing this method with 6 arguments and duplicating only a little bit of code is worth considering?